src/object/srv_obj.c

```diff
@@ -383,6 +383,11 @@ ds_obj_update_sizes_in_reply(crt_rpc_t *rpc)
 	iods = orw->orw_iods.ca_arrays;
 	size_count = orw->orw_iods.ca_count;
 
+	D_ASSERT(size_count > 0);
```

Comment:
same question as below. should we assert here?

Reply:
I don't think we can do this, it means client passed in unexpected parameters (data error), if you really want to detect if this can happen or not, please add the assert on client.